.github/workflows/test-short.yml

```
@@ -0,0 +1,20 @@
+name: test-short
+on:
+  pull_request:
+
+env:
+  GO_VERSION: '1.21'
+
+jobs:
+  test-short:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Run tests
```

once we move this to test.yml, proposal to disambiguate the job names by something like

```suggestion
      - name: Run tests in short mode
```

---

Updated! Thanks